src/lib/ruby-to-blocks-converter/gdx_for.js

```diff
@@ -11,7 +11,7 @@ const GdxForConverter = {
         if ((this._isSelf(receiver) || receiver === Opal.nil) && !rubyBlock) {
             switch (name) {
             case 'gdx_for_acceleration':
-                if (args.length === 1) {
+                if (args.length === 1 && this._isString(args[0])) {
```

Comment:
ここはね。args[0]に変数のブロックも入ってくるため、 this._isStringOrBlock(args[0]) じゃないといけないんだ。

```suggestion
                if (args.length === 1 && this._isStringOrBlock(args[0])) {
```